src/components/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { BiMenuAltRight } from "react-icons/bi";
+import { useAuth } from "../contexts/auth-context/authContext";
+
+function Header() {
+  const { currentUser } = useAuth();
+
+  return (
+    <div>
+      <div className="flex justify-between items-center h-20">
+        <div className="px-3 py-2.5 bg-green-500 text-white rounded-full">
+          {currentUser?.firstName[0]}
```

Comment:
why firstName[0] ?

Reply:
It's kind of profile picture which we see in like Gmail and all with firstname and last name when we don't upload photo